packages/cosmopolitan/cosmopolitan.0.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors: "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage: "https://github.com/dinosaure/esperanto"
+bug-reports: "https://github.com/dinosaure/esperanto/issues/"
+license: "MIT"
+tags: "org:mirage"
+dev-repo: "git+https://github.com/dinosaure/esperanto.git"
+build: [
+  [ "sh" "-c" "cd toolchain ; ./configure.sh --prefix=%{prefix}%" ]
+  [ "sh" "-c" "cd toolchain ; %{make}% all V=1" ]
+]
+# We should depend on conf-binutils but the package does not work on FreeBSD
+# even if we can do [pkg install binutils] on FreeBSD 12
+# depends: [
+#   "conf-binutils"
+# ]
+install: [ "sh" "-c" "cd toolchain ; %{make}% install" ]
+synopsis: "Cosmopolitan toolchain for OCaml compiler"
+description: "A little toolchain for OCaml with Cosmopolitan"
+url {
+  src: "https://github.com/dinosaure/esperanto/releases/download/v0.0.1/esperanto-v0.0.1.tar.gz"
```

Comment:
If it’s possible I’d prefer if cosmopolitan was taken directly from upstream instead of having a fork here.
If anyone else wants to use cosmopolitan in opam, they would expect to have the upstream version.

Reply:
So to clarify a bit, the distribution `dinosaure/esperanto` includes:
1) the "C toolchain" with Cosmopolitan 2.0.1: it's a like an _opaminication_ of _upstream_ plus few scripts and files (such as files to include) which are needed by `esperanto`
2) the OCaml toolchain (in the `ocamlfind`'s sense) with Cosmopolitan

If you are worried by name and the current layout of `esperanto`, I can rename the cosmopolitan package to something else and let the place free for someone who wants to "just" install Cosmopolitan _upstream_. But it's hard for me to decoupling `esperanto` and my `cosmopolitan` package from the distribution point-of-view (when they are highly dependant).